### PR TITLE
Added support for warszawa19115.pl

### PR DESCRIPTION
**Pull request by FliesWithWind**
Added support for Warsaw, Poland https://warszawa19115.pl/harmonogramy-wywozu-odpadow

I based on Banyule City Council integrations, so big thanks to the author!

### CODE REVIEW

File: custom_components/waste_collection_schedule/waste_collection_schedule/source/warszawa19115_pl.py [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+
+    @property
+    def geolocation_id(self) -> str:
+        if self._geolocation_id is None:
+            payload='_portalCKMjunkschedules_WAR_portalCKMjunkschedulesportlet_INSTANCE_o5AIb2mimbRJ_name=' + self._street_address
+            geolocation_session = requests.Session()
+            geolocation_request = geolocation_session.get(self.OC_SESSION_URL)
+            geolocation_request.raise_for_status()
+            headers = {
+            'sec-ch-ua': '" Not A;Brand";v="99", "Chromium";v="101", "Google Chrome";v="101"',
```
Comment (mampfes) [inline custom_components/waste_collection_schedule/waste_collection_schedule/source/warszawa19115_pl.py:58]: Please remove optional header fields and only keep the really mandatory fields (typically User-Agent). Move common header fields for the 2 requests to a single global variable.

File: custom_components/waste_collection_schedule/waste_collection_schedule/source/warszawa19115_pl.py [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+    pass
+
+
+class SourceParseError(ValueError):
+    pass
+
+
+
+class Source:
+    OC_GEOLOCATION_SEARCH_URL = 'https://warszawa19115.pl/harmonogramy-wywozu-odpadow?p_p_id=portalCKMjunkschedules_WAR_portalCKMjunkschedulesportlet_INSTANCE_o5AIb2mimbRJ&p_p_lifecycle=2&p_p_state=normal&p_p_mode=view&p_p_resource_id=autocompleteResourceURL&p_p_cacheability=cacheLevelPage&p_p_col_id=column-1&p_p_col_count=1'
```
Comment (mampfes) [inline custom_components/waste_collection_schedule/waste_collection_schedule/source/warszawa19115_pl.py:36]: Please use requests `params` argument to pass parameters in urls.

**Comment by FliesWithWind** [comment]
Thanks for the review. Updated :)

**Comment by mampfes** [comment]
Thanks for your contribution.

**Pull request merged**